static/src/javascripts/projects/commercial/modules/sticky-mpu.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import fastdom from 'lib/fastdom-promise';
+import { Sticky } from 'common/modules/ui/sticky';
+import { register, unregister } from 'commercial/modules/messenger';
+
+const noSticky =
+    document.documentElement &&
+    document.documentElement.classList.contains('has-no-sticky');
+let stickyElement;
+let rightSlot;
+
+const onResize = (specs, _, iframe) => {
+    if (rightSlot.contains(iframe)) {
+        unregister('resize', onResize);
+        stickyElement.updatePosition();
+    }
+};
+
+const stickyMpu = (adSlot: any) => {
+    if (adSlot.getAttribute('data-name') !== 'right') {
```

Comment:
Yeah it should be, but I couldn't get the assignment to `adSlot.parentNode.style.height` (line 48) to be happy with that type setting.